pkgs/development/libraries/qt-6/modules/qtbase.nix

```
@@ -202,7 +211,7 @@ stdenv.mkDerivation rec {
   inherit patches;
 
   # https://bugreports.qt.io/browse/QTBUG-97568
-  postPatch = ''
+  postPatch = lib.optionalString (!stdenv.hostPlatform.isMinGW) ''
```

```suggestion
  postPatch = ''
```
`coreutils` are executed on the build platform not the host platform.